monkey/tests/unit_tests/infection_monkey/puppet/test_plugin_source_extractor.py

```diff
@@ -13,8 +13,10 @@
 def build_agent_plugin(source_tar_path: Path, name="test_plugin") -> AgentPlugin:
     # We're using construct() here because we want to be able to construct plugins with invalid
     # names. Specifically, this is used for in test_plugin_name_directory_traversal()
-    manifest = AgentPluginManifest.construct(name=name, plugin_type=AgentPluginType.EXPLOITER)
-    return AgentPlugin.construct(
+    manifest = AgentPluginManifest.model_construct(
+        name=name, plugin_type=AgentPluginType.EXPLOITER, title=None
+    )
+    return AgentPlugin.model_construct(
```

Comment:
Do we need to call model_construct()? Can't we just call the constructor? Do we still need this?